src/Illuminate/Database/Console/Migrations/FreshCommand.php

```diff
@@ -57,6 +59,10 @@ public function handle()
             $this->runSeeder($database);
         }
 
+        $this->laravel[Dispatcher::class]->dispatch(
+            new DatabaseRefreshed()
+        );
+
```

Comment:
Would it be possible to move this so it dispatches _before_ the seeders are being run? Or have a separate event or something for that? It will run at different times now depending on whether or not you run `migrate:fresh --seed` or `migrate:fresh && db:seed`